Tasks/HelmDeployV0/task.loc.json

```diff
@@ -29,6 +29,12 @@
       "displayName": "ms-resource:loc.group.displayName.commands",
       "isExpanded": true
     },
+    {
+      "name": "azureContainerRegistry",
+      "displayName": "ms-resource:loc.group.displayName.azureContainerRegistry",
+      "isExpanded": true,
+      "visibleRule": "command == save"
```

Comment:
If save command takes a chart as the input, we should update the visibleRule for chartType input also like this.

Reply:
Updated the visibility rules and added 2 missing input fields. Also as discussed offline, updated the input variable names to avoid confusion with the variables for install command.